static/src/javascripts/projects/common/modules/analytics/scrollDepth.spec.js

```diff
@@ -11,7 +11,8 @@ jest.mock('lib/mediator');
 
 jest.useFakeTimers();
 
-describe('Scroll depth', () => {
+// TODO: remove skip
```

Comment:
maybe the  [`// #?`](https://github.com/guardian/frontend/wiki/So-you-want-to-ES6%3F#%EF%B8%8F-weirdness) rather than `TODO`? not a blocker though